package.json

```diff
@@ -31,6 +31,7 @@
     "express": "^4.17.1",
     "express-handlebars": "^5.3.4",
     "express-session": "^1.17.2",
+    "lint": "^0.7.0",
```

Comment:
We are already using eslint? Not sure this is needed, especially as a regular dependency.
You can uninstall using `npm uninstall lint`

Reply:
I didnt do any of this lol